File: Assignments/Assignment_17/p7.c
#include<stdio.h>
void main(){

        int row;
        printf("Enter the no of rows : \n");
        scanf("%d",&row);
        for(int i=1;i<=row;i++){

                for(int j=1;j<=i;j++){

                        if(i%5==0){

                                printf("5\t");
                        }else if(i%2!=0){

                                printf("1\t");
                        }else{

                                printf("0\t");
                        }
                }printf("\n\n");
        }
}

File: Flutter/Assignment/Square pattern set-2/p7.dart
import "dart:io";
void main(){
	print("Enter the no. of rows:");
	int? row=int.parse(stdin.readLineSync()!);
	int x=1;
	for(int i=1;i<=row;i++){
		int y=1;
		for(int j=1;j<=x;j++){
			stdout.write("${y}\t");
			y++;
		}
		x+=2;
		print("");
	}
}
